src/scenes/level_scene.py

```diff
@@ -303,6 +312,9 @@ def load_level_content(self) -> None:
                         player = loader.init_player(player_el["name"])
                         player.position = player_el["position"]
                         self.players.append(player)
+            if self.directory != "maps/level_0/":
```

Comment:
Sorry I noticed this one only later.
I think it's wrong to rely on system path this way ; maybe one day the maps would be move to another folder, or the levels be renamed. I know it's unlikely, but still.

My advise would be to rely on the attribute `self.number` instead, I think it should represent the id of the level, so if it's 0, then no need to open the save dialog because it means it's the first level.